test/meetup.spec.js

```diff
@@ -121,6 +133,80 @@ describe('Meetups List Api Exists', () => {
         .send(params)
         .end((err, res) => {
           expect(res.statusCode).to.equal(400);
+          expect(res.body).to.be.an('object');
+          expect(res.body.message).to.equal('title is required');
+          done();
+        });
+    });
+
+    it('should fail on POST with title length less than 5', (done) => {
+      params = {
+        title: 'me',
+        location: 'lagos',
+        happeningOn: '22-04-2020',
+        tags: 'apple',
+      };
+      request(server)
+        .post('/api/v1/meetups')
+        .send(params)
+        .end((err, res) => {
+          expect(res.statusCode).to.equal(400);
+          expect(res.body).to.be.an('object');
+          expect(res.body.message).to.equal('title length must be greater than 5');
+          done();
+        });
+    });
+
+    it('should fail on POST with location length less than 3', (done) => {
+      params = {
+        title: 'javascript',
+        location: 'la',
+        happeningOn: '22-04-2020',
+        tags: 'apple',
+      };
+      request(server)
+        .post('/api/v1/meetups')
+        .send(params)
+        .end((err, res) => {
+          expect(res.statusCode).to.equal(400);
+          expect(res.body).to.be.an('object');
+          expect(res.body.message).to.equal('location length must be greater than 3');
+          done();
+        });
+    });
+
+    it('should fail on POST with tag length less than 3', (done) => {
+      params = {
+        title: 'javascript',
+        location: 'los angeles',
+        happeningOn: '22-04-2020',
+        tags: 'ap',
+      };
+      request(server)
+        .post('/api/v1/meetups')
+        .send(params)
+        .end((err, res) => {
+          expect(res.statusCode).to.equal(400);
+          expect(res.body).to.be.an('object');
+          expect(res.body.message).to.equal('tags length must be greater than 3');
+          done();
+        });
+    });
+
+    it('should fail on POST with date input', (done) => {
```

Comment:
I think this should be incorrect date input

Reply:
alright, I'm going to fix that